js/app/directives/colorpickerDirective.js

```diff
@@ -21,12 +21,65 @@
  *
  */
 
+// expected hue range: [0, 360)
+// expected saturation range: [0, 1]
+// expected lightness range: [0, 1]
+var hslToRgb = function(hue, saturation, lightness){
+  // based on algorithm from http://en.wikipedia.org/wiki/HSL_and_HSV#Converting_to_RGB
+  if( hue == undefined ){
```

Comment:
Wait a minute, the entire project uses space and not tabs, should I change the whole file @georgehrke ?